### DISCUSSION TITLE
How do I change the initial conversation mode for the web UI app?

### DISCUSSION THREAD

**Discussion by gerroon** [cheshire-cat-ai/core#637] [Q&A] [answered]
HI

The conversation style is way too playful for my need. Is there a way to change the interaction mode for the web app? I am using local Ollama llama2

thanks

**Comment by pieroit** [comment, marked as answer]
Install the C.A.T. plugin (cat advanced tools), it lets you change the system prompt easily

Or write a plugin and hook up the `agent_prompt_prefix` hook
Tutorial on your first plugin both in docs and blog 

Please avoid using issues for personal support requests, come to the discord... Here we develop the framework

**Reply by gerroon** [reply]
Thanks, I think I did it under "Discussions QA" tab instead of issues. But I will make sure next time.
